extensions/jupyter-adapter/src/JupyterMessageDeserializer.ts

```diff
@@ -58,9 +58,7 @@ export function deserializeJupyterMessage(message: any[], key: string, channel:
 		parent_header: JSON.parse(parent), // eslint-disable-line
 		metadata: JSON.parse(metadata),
 		content: JSON.parse(content),
-
-		// TODO: Do we need to pass buffers?
-		buffers: []
+		buffers: message.slice(5),
```

Comment:
Maybe a comment about what/ why the slice is doing here? I'm assuming it's taking off some prefix?